lib/rich_text.rb

```diff
@@ -86,7 +86,7 @@ def linkify(text, mode = :urls)
 
   class HTML < Base
     def to_html
-      linkify(sanitize(simple_format(self)))
+      linkify(sanitize(simple_format(self, dir: "auto")))
```

Comment:
[This helper](https://api.rubyonrails.org/classes/ActionView/Helpers/TextHelper.html#method-i-simple_format) is not called directly. Look at `def simple_format(text)` above in this file, it doesn't have extra arguments.

Reply:
Yup, that's what I thought. How about this? (commit I just pushed)